src/ralph/data_center/admin.py

```diff
@@ -283,7 +283,7 @@ class DataCenterAssetOperation(OperationViewReadOnlyForExisiting):
 
 
 class DataCenterAssetSecurityInfo(SecurityInfo):
-    url_name = 'datacenter_asset_security_info'
+    url_name = 'security_info'
```

Comment:
ok, then i can revert this rename thing.. and create for example some map dict or something, it's productive however could be more errorproof..
what do you think?